fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/segmentedcontrols/Pill.kt

```diff
@@ -190,9 +205,25 @@ fun PillButton(
                     )
                 }
 
-                if (badge != null) {
-                    // TODO: Add logic to display badge
-                    Surface(content = badge)
+                if(pillMetaData.notificationDot) {
+                    val notificationDotColor: Color = getPillButtonTokens().notificationDotColor(getPillButtonInfo()).getColorByState(
+                        enabled = pillMetaData.enabled,
+                        selected = pillMetaData.selected,
+                        interactionSource = interactionSource
+                    )
+                    Spacer(Modifier.requiredWidth(GlobalTokens.size(GlobalTokens.SizeTokens.Size20)))
+                    Canvas(
+                        modifier = Modifier
+                            .padding(top = 2.dp, bottom = 12.dp)
+                            .sizeIn(minWidth = 6.dp, minHeight = 6.dp)
+                    ) {
+                        drawCircle(
+                            color = notificationDotColor, style = Fill, radius = dpToPx(3.dp)
+                        )
+                    }
+                    Spacer(Modifier.requiredWidth(GlobalTokens.size(GlobalTokens.SizeTokens.Size100)))
```

Comment:
use Size80 token here

Reply:
Fixed with actual conditions. In case of icon 100 was needed but for text 80 is needed.